src/components/media_manager/src/file_streamer_adapter.cc

```diff
@@ -85,7 +85,8 @@ void FileStreamerAdapter::FileStreamer::Disconnect() {
     delete file_stream_;
     file_stream_ = NULL;
   }
-  file_system::DeleteFile(file_name_);
+  // TODO: (malirod) uncomment. Prevents QT build
```

Comment:
Do we have the requirement for this?
Deleting file fixed the Blocker defect APPLINK-4700